test/controllers/actions/DataRetrievalActionSpec.scala

```diff
@@ -17,21 +17,27 @@
 package controllers.actions
 
 import base.SpecBase
+import cats.data.EitherT
+import errors.TrustErrors
+import handlers.ErrorHandler
 import models.UserAnswers
 import models.requests.{IdentifierRequest, OptionalDataRequest}
 import org.mockito.MockitoSugar
+import org.scalatest.EitherValues
 import org.scalatest.concurrent.ScalaFutures
+import play.api.mvc.Result
 import repositories.SessionRepository
 import uk.gov.hmrc.auth.core.AffinityGroup.Organisation
 import uk.gov.hmrc.auth.core.retrieve.Credentials
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-class DataRetrievalActionSpec extends SpecBase with MockitoSugar with ScalaFutures {
+class DataRetrievalActionSpec extends SpecBase with MockitoSugar with ScalaFutures with EitherValues {
```

Comment:
can this be renamed to DataRetrievalRefinerActionSpec to reflect the controller name